etc/db_schema.xml

```diff
@@ -26,6 +26,7 @@
         <column xsi:type="int" name="id" unsigned="true" nullable="false" identity="true" comment="ID"/>
         <column xsi:type="int" name="quote_id" unsigned="true" nullable="false" identity="false" comment="Magento Quote ID"/>
         <column xsi:type="smallint" name="order_created" default="0" comment="Magento Order Should Be Created On Magento Side"/>
+        <column xsi:type="varchar" name="flow_type" nullable="true" comment="Bold Flow Type"/>
```

Comment:
Should this column have a size defined, or do we just want to use the default of 255?